cluster/gce/gci/configure-helper.sh

```diff
@@ -341,6 +355,15 @@ function start-kubelet {
       flags+=" --hairpin-mode=${HAIRPIN_MODE}"
     fi
   fi
+  # Network plugin
+  if [[ -n "${NETWORK_PROVIDER:-}" ]]; then
+    flags+=" --network-plugin-dir=/home/kubernetes/bin"
```

Comment:
Go ahead then. I have not started. Please also refer to the discussions in #28563.